.github/workflows/main.yml

```diff
@@ -122,7 +122,7 @@ jobs:
       - name: Build android_rpc
         working-directory: apps/android_rpc
         run: |
-          export PATH="${ANDROID_NDK_HOME}:$PATH"
+          export PATH="${ANDROID_NDK_LATEST_HOME}:$PATH"
```

Comment:
Can you also add `set -eux` to all the inline scripts so we get errors about unbound variables if something changes in the future?